pages/api/content/glamorous.js

```diff
@@ -1,6 +1,131 @@
 module.exports = {
-  title: '`glamorous`',
+  title: 'glamorous API',
   subtitle: '',
-  description: ``.replace(/~/g, '`'),
+  description: `
+    The ~glamorous~ function allows you to create your own ~glamorousComponentFactory~ (see 
+    [above](https://github.com/paypal/glamorous/blob/master/README.md#glamorouscomponentfactory)
```

Comment:
I agree that we need a change, but I think having the API first is fine. So that means we just need to update this line to be `see below` so that it's accurate to the website's order.

@tdeschryver?